Assets/Mirror/Runtime/RemoteCallHelper.cs

```diff
@@ -32,6 +32,12 @@ public static class RemoteCallHelper
     {
         static readonly Dictionary<int, Invoker> cmdHandlerDelegates = new Dictionary<int, Invoker>();
 
+        [RuntimeInitializeOnLoadMethod]
```

Comment:
also, this [Runtimeinit] + Reset() method might not be obvious 5 years from now.
I think it's worth adding a comment on each one like
// support fast enter playmode without domain reload

Reply:
Changes to RemoteCallHelper were reverted as not needed.